server/src/graphql.ts

```diff
@@ -20,7 +23,16 @@ export const createApolloServer = async function (app: Express, config: Config)
 
     const typeDefs = loadSchemaFiles(join(__dirname, '/schema/')).join('\n');
     const schema = buildSchema(typeDefs, { assumeValid: true });
-    const context = createOffixMongoCRUDRuntimeContext(models, schema, db, pubSub);
+
+    const context = createKeycloakRuntimeContext({ 
```

Comment:
cc @wtrocki I added this helper method to the auth package. It's kinda based on the idea from a previous PR that was doing some refactors to the Graphback runtime. It allows you to pass in the uninitialised DataProvider and CrudService class and the helper will create them for you. You can see the auth package PR for how it works. What do you think of this approach?


Going with an approach like this potentially removes the need for different helper functions for all the different DataProvider classes given they all do the exact same thing. 

To make it really solid I think we'd have to refactor the constructors a little bit in the existing DataProvider and CrudService classes to accept a single options object (that can be extended) instead of multiple arguments.

Reply:
> To make it really solid I think we'd have to refactor the constructors a little bit in the existing DataProvider and CrudService classes to accept a single options object (that can be extended) instead of multiple arguments.

I would love to do it with you but we want to deliver this feature first and I do not see this as blocker. I have a couple issues to resolve before we need to go with. Since we are working on CRUDspec now on feature branch it will be unwise to start refactoring this. 

PR's that are blocking:
CRUD feature branch
Issues that are blocking this
https://github.com/aerogear/graphback/issues/1159
https://github.com/aerogear/graphback/issues/1157
https://github.com/aerogear/graphback/issues/1158

Reply:
We not going to get this solid until we try this out in the regular use cases so we are happy to get something out there - this will be alpha anyway.